x/ecocredit/server/basket/msg_update_basket_fee.go

```diff
@@ -0,0 +1,34 @@
+package basket
+
+import (
+	"context"
+
+	v1beta1 "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	basketv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/basket/v1"
+	baskettypes "github.com/regen-network/regen-ledger/x/ecocredit/basket"
```

Comment:
We should prefix non-native packages with the name of the package. Using `v1beta1` implies `v1beta1` of the basket package. We could use `v1` rather than `basketv1` and `sdkv1beta1` rather than `v1beta1`.